apps/cookbook/src/app/app.routes.ts

```diff
@@ -31,11 +31,11 @@ export const routes: Routes = [
       },
       {
         path: 'showcase',
-        loadChildren: () => import('./showcase/showcase.module').then((m) => m.ShowcaseModule),
+        loadChildren: () => import('./showcase/showcase.routes').then((m) => m.SHOWCASE_ROUTES),
```

Comment:
This seems to be the modern way of lazy-loading multiple sub-routes in a standalone context.

Reply:
Only based on the [older v17 docs](https://v17.angular.io/guide/standalone-components#lazy-loading-many-routes-at-once) 🤷 Cannot seem to find it in the official docs.